applications/NXsastof.nxdl.xml

```diff
@@ -27,6 +27,21 @@
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
   >
+  <symbols>
+    <doc>
+	  The symbol(s) listed here will be used below to coordinate datasets
+	  with the same shape.
+	</doc>
+	<symbol name="nXPixel">
```

Comment:
apply consistent names and descriptions here (see NXreftof for similar comment)

Reply:
see https://github.com/nexusformat/definitions/pull/819#discussion_r510940391